### ISSUE TITLE
Example wanted: with native i18n

### ISSUE THREAD

**Issue by FDiskas** [getsaf/shallow-render#178]
I'm using import https://angular.io/guide/i18n to localize my app

And using simple test case i'm getting error
```
The ANALYZE_FOR_ENTRY_COMPONENTS token only supports 'multi = true'!
```

**Comment by getsaf** [comment]
It's hard to analyze your issue without an example. Just googling that token ([`ANALYZE_FOR_ENTRY_COMPONENTS`](https://angular.io/api/core/ANALYZE_FOR_ENTRY_COMPONENTS)), it looks like it's related to routing.

My best guess is that your test fails because one of your modules is not setup properly. I would recommend checking your module setup and making sure that if you do use the `ANALYZE_FOR_ENTRY_COMPONENTS` token, that you have the `multi` flag enabled.

If that doesn't pan out, could you post your component/module/test here?

**Comment by FDiskas** [comment]
I added you to the private repo. To check it out. Pls do not push anything to that reppo

And thanks for the help ant for your lib

**Comment by getsaf** [comment]
Thanks, I was able to reproduce the issue and it looks like it's a gap in shallow-render support for mocking certain `multi` Inspection Tokens.

I've got a branch to fix that issue. Need to tend to my day-job so I probably won't get to cutting a new release until later in the day.

**Comment by Exomnius** [comment]
@getsaf Got the same issue in one of my projects. If possible, can you push your branch with the fix? Then I can use/test the fix.

**Comment by getsaf** [comment]
I just pushed with a fix for the `'multi = true'` error. Please try [10.1.3](https://github.com/getsaf/shallow-render/releases/tag/v10.1.3)

**Comment by Exomnius** [comment]
Could it be that this fix results in the "Component [COMPONENT] is not part of any NgModule or the module has not been imported into your module." issue again? I see that it was previously fixed in 9.0.3

**Comment by getsaf** [comment]
> Could it be that this fix results in the "Component [COMPONENT] is not part of any NgModule or the module has not been imported into your module." issue again? I see that it was previously fixed in 9.0.3

Closing this issue, see #185 for the NgModule error fix.
